extensions/tasks/NLUTestV0/index.ts

```diff
@@ -2,12 +2,14 @@ import * as tl from 'azure-pipelines-task-lib/task';
 import * as tr from 'azure-pipelines-task-lib/toolrunner';
 import * as path from 'path';
 import * as utilities from 'nlu-devops-common/utilities';
+import { publishTestResults } from './publishTestResults';
 
 async function run() {
     try {
         const output = getOutputPath();
         await runNLUTest(output);
         await runNLUCompare(output);
+        publishTestResults(['**/TestResult.xml'], tl.getVariable('Agent.TempDirectory'));
```

Comment:
```
const shouldPublishTestResults = tl.getBooleanInput('publishTestResults');
if (shouldPublishTestResults) {
  publishTestResults(...)
}
```

Reply:
Or - to be consistent with `runNLUCompare`, move the check into `publishTestResults` and make it a no-op.